cli/dcoscli/marathon/main.py

```diff
@@ -32,11 +32,17 @@ def _main(argv):
         argv=argv,
         version='dcos-marathon version {}'.format(dcoscli.version))
 
-    return cmds.execute(_cmds(), args)
+    marathon_client = marathon.create_client()
+    pod_subcommand = MarathonPodSubcommand(resource_reader=_get_resource,
+                                           marathon_client=marathon_client)
```

Comment:
Moved the initialization of `MarathonPodSubcommand` to the `_main` method, so that it does not happen when the module is imported. I just noticed another potential issue though, which is that `marathon.create_client()` will be called more than once if a non-pods command is being run. I have some ideas for how to solve it, but maybe it's not a big issue?